samples/cpp/benchmark_app/main.cpp

```diff
@@ -337,6 +358,8 @@ int main(int argc, char* argv[]) {
     try {
         ov::CompiledModel compiledModel;
 
+        model_sentry releaser(compiledModel);
```

Comment:
Will `release_memory()` pass if the exception thrown before `compiledModel` is initialized?